dataland-frontend/src/components/forms/CreateP2pDataset.vue

```diff
@@ -264,6 +292,34 @@ export default defineComponent({
         this.postP2pDataProcessed = true;
       }
     },
+
```

Comment:
Can someone explain these changes to me?

Reply:
They are necessary to have this page also upload documents. You can call me for further explanation.

Reply:
Bugs were not encountered on the deployed version. So this seems to work and iam closing this here